git-date/tests/time/parse.rs

```diff
@@ -12,3 +13,43 @@ fn special_time_is_ok_for_now() {
         }
     );
 }
+
+#[test]
+fn short() {
+    assert_eq!(
+        git_date::parse("1979-02-26"),
+        Some(Time {
+            seconds_since_unix_epoch: 288835200,
+            offset_in_seconds: 0,
+            sign: Sign::Plus,
+        }),
+        "could not parse with SHORT format"
+    );
+}
+
+#[test]
+fn rfc2822() {
+    assert_eq!(
+        git_date::parse("Thu, 18 Aug 2022 12:45:06 +0800"),
+        Some(Time {
+            seconds_since_unix_epoch: 1660797906,
+            offset_in_seconds: 28800,
+            sign: Sign::Plus,
+        }),
+        "could not parse with RFC2822 format"
+    );
+}
+
+#[test]
+fn relative() {
+    let two_weeks_ago = git_date::parse("2 weeks ago").expect("valid time");
```

Comment:
These tests will be easier to write once `parse(…)` takes the current time and is 'pure'.